### DISCUSSION TITLE
hi tsyuay, i wish you would add sand castle, it's my favorite game.

### DISCUSSION THREAD

**Discussion by demonster2023** [Tsyuay/FrivReborn#44] [Q&A] [answered]
I wanted you to put the keyboard buttons. <img width="1541" alt="sand castle" src="https://user-images.githubusercontent.com/121765699/215343641-355efc7f-01a4-47bc-aff8-3bcdddabfb7c.png">

swf game:https://web.archive.org/web/20191002231546/http://l.funo.com/z/sandcastle/sandcastle.swf

walkthrough:https://web.archive.org/web/20201230235804/https://www.friv.com/x/walkthroughs/sandcastle.html

follow the links to add the game and walkthrough.

**Comment by Tsyuay** [comment, marked as answer]
done
